### ISSUE TITLE
dnslink.info does not exist

### ISSUE THREAD

**Issue by Kubuxu** [ipfs/go-dnslink#2]
It is mentioned in readme.

**Comment by RichardLitt** [comment]
@lgierth Who owns dnslink.info?

**Comment by ghost** [comment]
Don't know -- somebody must have grabbed it.

> Creation Date: 2015-12-12T12:08:55Z

**Comment by RichardLitt** [comment]
Strangely, there's no whois info.

**Comment by RichardLitt** [comment]
Is there another domain we could use? Was there a domain we were supposed to be pointing at?

**Comment by ghost** [comment]
> Strangely, there's no whois info.

Using WhoisGuard is pretty common, all big registrars offer it and with some it's even the default.

> Is there another domain we could use? Was there a domain we were supposed to be pointing at?

We could just go with dnslink dot io

**Comment by ghost** [comment]
We have dnslink.io now

**Comment by RichardLitt** [comment]
Cool. What should we put there? It says 'see dnslink.info'. What was planned for that website?

**Comment by ghost** [comment]
Nothing from what I can tell. Someone needed an example domain in the documentation and went with dnslink.info instead of using the standard example domains example.net, example.com, etc.

We can put a website there but I don't think it's very urgent. If we want the domain to resolve quickly, we can have it redirect to this repo.

**Comment by RichardLitt** [comment]
Ah, so we got it so that people could use it as a test site. Great. Maybe just put up a static text document?

**Comment by RichardLitt** [comment]
Should we update the text from dnslink.info to dnslink.io?

**Comment by ghost** [comment]
Yeah and we should add the respective TXT record so the examples work

**Comment by lidel** [comment]
(quick note) we should replace all uses of `dnslink.info` and `dnslink.io` with `dnslink.dev`
